envisage/ui/single_project/project.py

```diff
@@ -718,4 +721,43 @@ def _location_changed(self, old, new):
         self.dirty = True
 
 
+# Helper functions
+
+# Added so that the dependency on this function from Traits can be removed
+def clean_filename(name, replace_empty=""):
+    """
+    Make a user-supplied string safe for filename use.
+    Returns an ASCII-encodable string based on the input string that's safe for
+    use as a component of a filename or URL. The returned value is a string
+    containing only lowercase ASCII letters, digits, and the characters '-' and
+    '_'.
+    This does not give a faithful representation of the original string:
+    different input strings can result in the same output string.
+    Parameters
```

Comment:
Looks like some of the formatting got lost in translation; please could you re-add the blank lines above the "Parameters" and "Returns" sections, and check that Sphinx is happy with the results?

Reply:
To check with Sphinx, is the proper way to do that to just run `python etstool.py docs`?

I did that and don't see any complaints that are related to the current changes.

Reply:
> is the proper way to do that to just run `python etstool.py docs`

Yep, that should do it.